src/vs/workbench/parts/debug/browser/debugEditorModelManager.ts

```diff
@@ -325,7 +325,7 @@ export class DebugEditorModelManager implements wbext.IWorkbenchContribution {
 
 	private static FOCUSED_STACK_FRAME_DECORATION: editorcommon.IModelDecorationOptions = {
 		isWholeLine: true,
-		className: 'debug-focussed-stack-frame-line',
+		className: 'debug-focused-stack-frame-line',
```

Comment:
Make sure you adopt the same change on `src/vs/workbench/parts/debug/browser/media/debug.contribution.css` where this class name is used.